docs/release-notes.d/00readme.md

```diff
@@ -0,0 +1,28 @@
+# Pending release notes directory
+
+This directory contains release note entries that have not been merged
+into the main release-notes.md document.
+
+Generally a release note entry should be created for changes that:
+
+- Fix bugs in the code.
+- Implement new features.
+- Change existing behavior.
+
+Release notes are generally not needed for:
+
+- Documentation improvements.
+- Strictly internal changes to the code that won't be visible to users
+  of the code.
+
+## Release note format
+
+Release notes are included in files under this `docs/release-notes.d`
+directory and have a name of `*.md`.  They will be included in the
+`release-notes.md` file, and should be formatted as a Markdown list
+entry.
+
+Choose a filename that is related to what this change does.  The names
+are not used for anything in particular, but to keep the files
```

Comment:
Can we establish that files are named by PR number that brings them or brings the change they are for? Otherwise we are in for a name conflict at some point anyway.

Reply:
It hasn't really been an issue with mbed TLS. Plus, you don't really know the pr until you push it.

Reply:
People can always amend commits once PR has been created...